packages/splitio-adapter/modules/adapter/adapter.js

```diff
@@ -86,14 +90,14 @@ const subscribe = ({ onFlagsStateChange, onStatusStateChange }) =>
   });
 
 const configure = ({
-  clientSideId,
+  authorizationKey,
   user,
   onFlagsStateChange,
   onStatusStateChange,
 }) => {
   adapterState.user = ensureUser(user);
-  const { client, manager } = initializeUserContext(
-    clientSideId,
+  const { client, manager } = initializeClient(
```

Comment:
This also is more of client inititialization to me to get the `client` and `manager`.